### ISSUE TITLE
nix-env -i on NixOS

### ISSUE THREAD

**Issue by edwtjo** [NixOS/nix#514]
I'm seeing:
```
error: derivation ‘/nix/store/hwj4kamrv8dkjy2qbcnh9wa0x8hycki6-user-environment.drv’ has ‘__noChroot’ set, but that's not allowed when ‘build-use-chroot’ is ‘true’
```
when I try to install stuff as a local user and not systemwide on nixos. I think this is wierd since I've set `nix.useChroot = true;` in `configuration.nix` and in `~/.nixpkgs/config.nix`. What am I doing wrong? I've tried removing the user-environment but to no avail.

I'm on `nix-env (Nix) 1.9pre4033_2be7f79`

**Comment by edolstra** [comment]
This was fixed in 6f0c6e20e03bc82fcf0d2198cf81fa2cf25c2f6c.

**Comment by edwtjo** [comment]
Great! Manually updating from hydra solved it. Just note that I'm using `nix.package = pkgs.nixUnstable` in `configuration.nix`. Perhaps it was a local thing since there have been a number of nixUnstable bumps since 2be7f79.
